tests/test_settings_api.py

```diff
@@ -584,3 +585,23 @@ def test_deprecated_command_arguments(mixing_elbow_case_data_session):
         "m2",
         "m3",
     }
+
+
+@pytest.mark.fluent_version(">=25.2")
+def test_return_types_of_operations_on_named_objects(mixing_elbow_settings_session):
+    solver = mixing_elbow_settings_session
+
+    var1 = solver.settings.setup.materials.fluid.create("air-created")
```

Comment:
We could also directly compare the objects:
```
assert solver.settings.setup.materials.fluid.create("air-created") == solver.settings.setup.materials.fluid["air-created"]

Reply:
Done.